### ISSUE TITLE
UX: faceted filtering is far too well hidden

### ISSUE THREAD

**Issue by sknorr** [gaois/terminologue#60]
It is really hard to find faceted filtering -- the only visual indications that it exists at all are:

1. There is a gray, non-descript panel to the left with a little hover effect. (Or on the bottom in narrow-screen mode.)
2. There is a screenshot of it in the help.

Please add an icon for this functionality to the side bar.

**Comment by michmech** [comment]
True, the UX isn't the best here. An icon or something would be handy.

This would actually be a feature request for [Screenful](https://github.com/michmech/screenful), the UI framework that Terminologue is using. The relationship between Terminologue and Screenful is explained [here](https://docs.gaois.ie/en/software/terminologue/source-code#screenful).

**Comment by sknorr** [comment]
I'd actually started to create a patch for this (as you can see from my commits above) -- 4b6d4b4 basically works but it is buggy: it looks a bit foreign in the toolbar, it's not always presenting state correctly, it's shown even in sidebars that actually have no facet filter, and I don't think it can be localized. If I get to fix all these things, I'd create a PR, if that is ok?

(My fork has a few more things, mostly CSS changes [we have too many main languages] and German l10n that I'd like to submit at some point.)
